### PR TITLE
change for time label

### PR DESCRIPTION
**Pull request by asylves1**
# Description

Add timestamp to output menu
image

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue [hunk 1]
```diff
@@ -68,6 +91,9 @@ const emit = defineEmits(['update:selection']);
 }
 
 .dropdown-option-group {
+	display: flex;
+	gap: 0.2rem;
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue:95]:
```suggestion
gap: var(--gap-xsmall);
```

File: packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue [hunk 2]
```diff
@@ -68,6 +91,9 @@ const emit = defineEmits(['update:selection']);
...
+	align-items: center;
 	font-size: var(--font-caption);
 	color: var(--gray-600);
 }
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue:99]: I think it'd be nice for the label and date to have space in between
```suggestion
   justify-content: space-between;
}
```

File: packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue [hunk 3]
```diff
@@ -31,14 +39,29 @@
...
+const getOptionById = (id, items) => items.find((option) => option.id === id);
+
+const getCreateTimeById = (id: string) => {
+	const items = props.options.flatMap((option) => option.items);
+	const option = getOptionById(id, items);
+	return getElapsedTimeText(option.timestamp);
+};
+
+const getLabelById = (id: string) => {
+	const items = props.options.flatMap((option) => option.items);
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue:61]: This is repeated in `getCreateTimeById`, maybe `items` should be a `computed` variable

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue (hunk 1 above)
Comment (shawnyama) [reply packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue:95]: Or I think it'd be nice to have some space in between
```suggestion
   justify-content: space-between;
```

Comment (asylves1) [reply packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue]: change made

File: packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue (hunk 3 above)
Comment (asylves1) [reply packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue]: changed to computed

**Pull request merged**